app/models/discourse_activity_pub_object.rb

```diff
@@ -61,14 +57,19 @@ def self.handle_model_callback(model, ap_type_sym)
 
       object.save!
 
-      if ap_type_sym != :update
-        DiscourseActivityPubActivity.create!(
+      unless ap_type_sym == :update && !model.activity_pub_published?
```

Comment:
Nitpicky, sorry, but I find it hard to read `unless` with multiple arguments, and especially when it's unless `not something`. 

This essentially means "log an activity if type is update and object is already published"?

Reply:
Not quite. It means log an activity unless you're dealing with an unpublished update (could be other types too, e.g. create). I feel that using an `if` would be more confusing here:

```
if ap_type_sym != :update || model.activity_pub_published?
....
```

Perhaps if the conditions are switched so they "read" in a grammatical fashion?

```
unless !model.activity_pub_published? && ap_type_sym == :update
```
i.e. unless you have an unpublished update (instead of unless you have an update that's not published)? I'll make that switch.